cli/src/commands/invite/greet.rs

```diff
@@ -61,6 +61,9 @@ pub async fn main(args: Args) -> anyhow::Result<()> {
             let ctx = step4_device(ctx).await?;
             step5_device(ctx).await
         }
+        InviteListItem::ShamirRecovery { .. } => {
+            todo!();
```

Comment:
Provide a clear error message for the user and add a TODO comment related to an issue